samples/add-map-image-layer/README.metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+    "category": "Layers",
+    "description": "Display a layer from an ArcGIS map image layer service.",
+    "formal_name": "AddMapImageLayer",
+    "ignore": false,
+    "images": [
+        "add-map-image-layer.png"
+    ],
+    "keywords": [
+        "display",
+        "image",
+        "layer",
+        "map",
+        "ArcGISMap",
+        "ArcGISMapImageLayer",
+        "MapView (Compose Toolkit)"
+    ],
+    "language": "kotlin",
+    "redirect_from": "TODO",
```

Comment:
```suggestion
```

Reply:
Kept it empty, otherwise git checker was complaining